frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -1304,8 +1375,12 @@ mod tests {
 			assert_eq!(
 				multi_phase_events(),
 				vec![
-					Event::SignedPhaseStarted { round: 1 },
-					Event::SolutionStored { compute: ElectionCompute::Signed, prev_ejected: false },
+					Event::PhaseTransition { from: Phase::Off, to: Phase::Signed, round: 1 },
```

Comment:
Event names should generally be passive 

```suggestion
					Event::PhaseTransitioned { from: Phase::Off, to: Phase::Signed, round: 1 },
```